sklearn/model_selection/_split.py

```diff
@@ -997,6 +997,16 @@ def split(self, X, y=None, groups=None):
             for train_index, test_index in cv.split(X, y, groups):
                 yield train_index, test_index
 
+    def get_n_splits(self):
+        """Returns the number of splitting iterations in the cross-validator
+
+        Returns
+        -------
+        n_splits : int
+            Returns the number of splitting iterations in the cross-validator.
+        """
+        return self.cvargs["n_splits"] * self.n_repeats
```

Comment:
I didn't get by what you mean by "bring present in charge", should I send the arguments -> n_splits and n_repeats in the get_n_splits function, because the get_n_splits for RepeatedKFold and RepeatedStratifiedKFold should return the multiplication of n_splits and n_repeats as those many splits are generated by them. Can you please tell what should I do.